### PR TITLE
Changing district to be represented by int

### LINKED ISSUE #35: District validation

### PR DESCRIPTION
**Pull request by tirameshu**

### CODE REVIEW

**Comment by tirameshu** [comment]
LGTM :D

**Comment by tirameshu** [comment]
All tests passed now!!

**Comment by tirameshu** [comment]
Resolves #35 #36

File: src/main/java/seedu/address/model/vehicle/District.java [hunk 1]
```diff
@@ -34,26 +28,21 @@ public District(String d) {
     /**
      * Returns true if a given string is a valid District.
      */
-    public static boolean isValidDistrict(String test) {
-        return test.matches(VALIDATION_REGEX);
+    public static boolean isValidDistrict(int test) {
+        return 1 <= test && test <= 28;
```
Comment (atharvjoshi) [inline src/main/java/seedu/address/model/vehicle/District.java]: Could you make these private static final variables? Right now, they seem like magic numbers! Thanks!

Comment (tirameshu) [reply src/main/java/seedu/address/model/vehicle/District.java]: Ok!

**Comment by atharvjoshi** [comment]
LGTM

**Pull request merged**